parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -389,8 +418,18 @@ async def file_create(
             fd = self.local_storage.create_file_descriptor(child) if open else None
 
         # Send events
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent.id)
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=child.id)
+        self._send_event(
```

Comment:
Could we have additional information? To add a file the GUI also needs its creation date, last update date, size, and if it's confined or not.

Reply:
entry_size is really needed ? I would say it's always 0 here given the file has just been created !

BTW I'm not really sure it's a good idea to go this way and add so much information in the event, typically here it means we should add `entry_created`&`entry_updated` to all the `FS_ENTRY_UPDATED` events and `entry_size` to the `FILE_WRITE` and `FILE_RESIZE` flavor (and it's pretty easy to forget to update all the events when a new field is needed...)

I think it's much cleaner to consider the event is just a trigger that notifies the GUI it should in turn go fetch the information it needs (so doing a `workspace_fs.path_info(current_folder / event.entry_name)`)